packages/core/cache/src/LMDBCache.js

```diff
@@ -111,27 +112,47 @@ export class LMDBCache implements Cache {
     return Buffer.concat(await Promise.all(buffers));
   }
 
-  async setLargeBlob(key: string, contents: Buffer | string): Promise<void> {
+  async setLargeBlob(
```

Comment:
Is there any elegant way to share the implementation between the two Cache types as (AFAICT) they're identical?

Reply:
Instantiate a `FSCache` instance inside lmdb cache and forward calls of `setLargeBlob` to that?

Reply:
That sounds good to me, will clean this up